paasta_tools/mesos/master.py

```diff
@@ -148,7 +183,7 @@ def resolve(self, cfg):
 
     @util.CachedProperty(ttl=15)
     def state(self):
-        return self.fetch("/master/state.json").json()
+        return self.fetch("/master/state.json", cached=True).json()
```

Comment:
I appreciate that we are making the cache opt-out, but doesn't this give no way for our other uses of this function to opt out?

In other words, isn't this always using the cache, and the --use-cache option which is much further removed has no effect?

Reply:
this arg is a way to tell _request "use caching if available". whether caching is available is defined by cache_enabled in MesosMaster.config: https://github.com/Yelp/paasta/pull/1556/files#diff-8a09a1f8585e7ecef8050b47ad0d189cR83